File: model/Event.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ETT.model
{
    public class Event
    {
        private string period { get; set; }
        //@JsonIgnore per mos me u gjeneru ne Json garant
        private int periodDay { get; set; }
        //@JsonIgnore
        private int periodTimeslot { get; set; }
        private string room { get; set; }

        public string getPeriod() {
		    return period;
	    }

        public void setPeriod(String period) {
		    this.period = period;
	    }

        public String getRoom() {
		    return room;
	    }

        public void setRoom(String room) {
		    this.room = room;
	    }

        public int getPeriodDay() {
		    return periodDay;
        }

        public void setPeriodDay(int periodDay) {
            this.periodDay = periodDay;
        }
        public Event() { }
        public Event(string period, int periodDay, int periodTimeslot, string room)
        {
            this.period = period;
            this.room = room;
            this.periodDay = periodDay;
            this.periodTimeslot = periodTimeslot;
        }
    public class Builder {
            private string period;
            //@JsonIgnore per mos me u gjeneru ne Json garant
            private int periodDay;
            //@JsonIgnore
            private int periodTimeslot;
            private string room;
            
            public Builder periodss(string period) {
                this.period = period;
                return this;
            }
            
            public Builder periodDays(int periodDay) {
                this.periodDay = periodDay;
                return this;
            }

            public Builder perTimeslots(int periodTimeslot) {
                this.periodTimeslot = periodTimeslot;
                return this;
            }

            public Builder rooms(string room) {
                this.room = room;
                return this;
            }
            
            
            public Event build() {
                return new Event();
            }
        }
        
    }
}

File: ExaminationTimetabling/src/main/java/com/timetabling/app/model/Event.java
package com.timetabling.app.model;

import com.fasterxml.jackson.annotation.JsonIgnore;
import com.fasterxml.jackson.annotation.JsonProperty;

public class Event {
	@JsonProperty("Period")
	public String Period;
	@JsonIgnore
	private Integer periodDay;
	@JsonIgnore
	private Integer periodTimeslot;
	@JsonProperty("Room")
	private String Room;
	
	public Event() {}
	
	@JsonIgnore
	public String getPeriod() {
		return Period;
	}

	@JsonIgnore
	public void setPeriod(String Period) {
		this.Period = Period;
	}

	@JsonIgnore
	public String getRoom() {
		return Room;
	}

	@JsonIgnore
	public void setRoom(String Room) {
		this.Room = Room;
	}
	
	@JsonIgnore
	public Integer getPeriodDay() {
		return periodDay;
	}

	@JsonIgnore
	public void setPeriodDay(Integer periodDay) {
		this.periodDay = periodDay;
	}

	@JsonIgnore
	public Integer getPeriodTimeslot() {
		return periodTimeslot;
	}

	@JsonIgnore
	public void setPeriodTimeslot(Integer periodTimeslot) {
		this.periodTimeslot = periodTimeslot;
	}

	public Event(Builder builder) {
		this.Period = builder.Period;
		this.Room = builder.Room;
		this.periodDay = builder.periodDay;
		this.periodTimeslot = builder.periodTimeslot;
	}
	
	public static class Builder {
		private String Period;
		private String Room;
		private Integer periodDay;
		private Integer periodTimeslot;
		
		public Builder Period(String Period) {
			this.Period = Period;
			return this;
		}
		
		public Builder Room(String Room) {
			this.Room = Room;
			return this;
		}
		
		public Builder periodDay(Integer periodDay) {
			this.periodDay = periodDay;
			return this;
		}
		
		public Builder periodTimeslot(Integer periodTimeslot) {
			this.periodTimeslot = periodTimeslot;
			return this;
		}
		
		public Event build() {
			return new Event(this);
		}
	}
}
